### ISSUE TITLE
Error while building stlc example

### ISSUE THREAD

**Issue by ulysses4ever** [QuickChick/QuickChick#57]
I have
```
File "./verif.v", line 403, characters 11-32:
Error: The reference NPeano.Nat.max_le_iff was not found in the current
environment.
```
when doing `make` inside `examples/stlc`, `master` branch. Coq 8.7.

**Comment by Lysxia** [comment]
There's now a fix in #58. :)

**Comment by ulysses4ever** [comment]
Thanks a lot!
